### ISSUE TITLE
pipewire permission issue after 1.0 release

### ISSUE THREAD

**Issue by shetozz** [flathub/io.mpv.Mpv#398]
In the flatpak permissions, pipewire is set to `xdg-run/pipewire-0:ro` this leads to audio not working if the system's pipewire version is 1.0.x, changing the permission to `xdg-run/pipewire-1:ro` solves the issue.

**Comment by Erick555** [comment]
> this leads to audio not working if the system's pipewire version is 1.0.x

what do you mean? socket number isn't related to pipewire version.

**Comment by shetozz** [comment]
Not sure what's going on, as I have no experience in any of this stuff, but that's what happened to me.

**Comment by Erick555** [comment]
do you still have pipewire-1 socket after reboot?

**Comment by shetozz** [comment]
Yeah, still `-1` even after restarting.

**Comment by Erick555** [comment]
Is there only `-1` or both `-1` and `-0`? This is very strange situation which would break all flatpaks that use pipewire. You may ask in https://gitlab.freedesktop.org/pipewire/pipewire/-/issues about it.

**Comment by shetozz** [comment]
I just found out something even weirder, it's not just `-1` that works, any number other than `-0` works, and even removing the permission altogether works, but resetting it to default breaks audio, -plays mp3 files, but audio in video files doesn't work-

**Comment by shetozz** [comment]
```css
 .dbus-proxy         app        gnome-shell     gvfsd       .mutter-Xwaylandauth.5TFDJ2        ICEauthority                pipewire-0.lock
 .flatpak            at-spi     gnupg           keyring     Alacritty-wayland-0-49215.sock     pipewire-0                  wayland-0
 .flatpak-cache      dconf      gsconnect       pulse       bus                                pipewire-0-manager          wayland-0.lock
 .flatpak-helper     doc        gvfs            systemd     gnome-session-leader-fifo          pipewire-0-manager.lock
```

**Comment by Erick555** [comment]
yeah, your socket name is `pipewire-0` which is right. By changing permission from `pipewire-0`to `pipewire-1` you actually blocked pipewire access which accidentally helped because the problem is pipewire didn't work at all with flatpak. There [were](https://github.com/flathub/io.mpv.Mpv/issues/376) similar issues about pipewire in the past.
